src/tests/unit/tests/common/components/cards/__snapshots__/instance-details.test.tsx.snap

```diff
@@ -1,305 +1,123 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`InstanceDetails isSelected drives the styling for the card 1`] = `
-<div
-  className="instanceDetailsCardContainer selected"
-  data-automation-id="instance-card"
->
+<DocumentFragment>
   <div
-    className="instanceDetailsCard selected interactive"
-    onClick={[Function]}
-    tabIndex={-1}
+    class="instanceDetailsCardContainer selected"
+    data-automation-id="instance-card"
   >
-    <div>
-      <table
-        className="reportInstanceTable"
-      >
-        <tbody>
-          <React.Fragment />
-          <React.Fragment />
-          <React.Fragment />
-        </tbody>
-      </table>
-      <button
-        aria-label="highlight test-id"
-        aria-pressed={true}
-        className="hiddenHighlightButton"
-        onBlur={[Function]}
-        onClick={[Function]}
-        onFocus={[Function]}
-      />
-      <InstanceDetailsFooter
-        deps={
-          {
-            "cardInteractionSupport": {
-              "supportsCopyFailureDetails": true,
-              "supportsHighlighting": true,
-              "supportsIssueFiling": true,
-            },
-            "getPropertyConfigById": [Function],
-          }
-        }
-        result={
-          {
-            "descriptors": {},
-            "identifiers": {
-              "conciseName": "test-concise-name",
-              "identifier": "test-id",
-              "this-property-does-not-have-config": "some value",
-            },
-            "isSelected": true,
-            "resolution": {},
-            "ruleId": "image-alt",
-            "status": "fail",
-            "uid": "some-guid-here",
-          }
-        }
-      />
+    <div
+      class="instanceDetailsCard selected interactive"
+      tabindex="-1"
+    >
+      <div>
+        <table
+          class="reportInstanceTable"
+        >
+          <tbody />
+        </table>
+        <button
+          aria-label="highlight test-id"
+          aria-pressed="true"
+          class="hiddenHighlightButton"
+        />
+        <mock-instancedetailsfooter
+          deps="[object Object]"
+          result="[object Object]"
+        />
+      </div>
```

Comment:
We should make sure we use the `expectMockedComponentPropsToMatchSnapshots` helper so we don't miss this information.

Reply:
On every InstanceDetails component render, after that i set below code, please review and let me know if you have any feedback.


const renderResult = render(<InstanceDetails {...props} />);
expectMockedComponentPropsToMatchSnapshots([InstanceDetailsFooter]);